packages/ia-wayback-search/stories/ia-wayback-search.stories.js

```diff
@@ -38,5 +38,13 @@ storiesOf('<ia-wayback-search>', module)
         waybackPagesArchived="32 trillion pages"
         waybackHost="archive.org"
       ></ia-wayback-search>
+      <ia-wayback-search
+        waybackPagesArchived="32 trillion pages"
+        waybackHost="archive.org"
+      >
+        <div slot="optional-branding">
+          <img src="https://archive.org/download/ia-25-wf/wf-wb-search-square.png">
```

Comment:
QUESTION: Same question as in other PR. Should we be hard-coding the production URL? Will that prevent easy testing on staging apps?